panther_analysis_tool/util.py

```diff
@@ -111,6 +111,26 @@ def func_with_backend(
     return lambda args: func(get_backend(args), args)
 
 
+def func_with_optional_backend(
+    func: Callable[[argparse.Namespace, Optional[BackendClient]], Any]
+) -> Callable[[argparse.Namespace], Tuple[int, str]]:
+    return lambda args: func(args, get_optional_backend(args))
+
+
+def get_optional_backend(args: argparse.Namespace) -> Optional[BackendClient]:
+    # The UserID is required by Panther for this API call, but we have no way of
+    # acquiring it, and it isn't used for anything. This is a valid UUID used by the
+    # Panther deployment tool to indicate this action was performed automatically.
+    user_id = "00000000-0000-4000-8000-000000000000"
```

Comment:
can we move this out into a constant that any package can utilize? I notice we duplicate this in a few spots

Reply:
updated the schema file with the latest and moved this to the constants!